Dockerfile.openshift

```diff
@@ -0,0 +1,23 @@
+# This dockerfile is specific to building the network metrics daemon for OpenShift
```

Comment:
is that comment a copy paste artifact? :)

Reply:
Ops (2).